pytorch_toolkit/tools/instantiate.py

```diff
@@ -14,23 +14,30 @@
 
 import argparse
 import glob
+import logging
 import os
 from subprocess import run
 
 import yaml
 
+def run_with_log(*args, **kwargs):
+    cmd = args[0]
+    logging.info(f'Running command\n`{cmd}`') #TODO: consider with Ilya
+    return run(*args, **kwargs)
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('destination')
     parser.add_argument('--do-not-load-snapshots', action='store_true')
+    parser.add_argument('--templates-pattern', default='**/template.yaml')
```

Comment:
Template and pattern do mean almost the same thing. I'd prefer `--templates-filter` or something alike.

Reply:
Fixed